### ISSUE TITLE
Sometimes clicked key even if corrent is always wrong!

### ISSUE THREAD

**Issue by Jirafey** [codicocodes/speedtyper.dev#154]
I am on Linux Debian 11 Xfce, I'm using brave browser.

I only typed for about 10 mins, this issue already happened twice. It happened with ')' key maybe it was due to too many failed attempts but this is just weird.

**Comment by codicocodes** [comment]
I don't think this is a bug, it is a lot more likely that you typed an  incorrect key or there was a previous incorrect character in the textarea that prevented you from typing a character.

I think the way to resolve this, is to enable the user to see exactly what is in the textarea so that they can confirm the issue themselves, instead of just showing an error as right now.

**Comment by Jirafey** [comment]
> I don't think this is a bug, it is a lot more likely that you typed an incorrect key or there was a previous incorrect character in the textarea that prevented you from typing a character.
> 
> I think the way to resolve this, is to enable the user to see exactly what is in the textarea so that they can confirm the issue themselves, instead of just showing an error as right now.
> 
> Also, confirming this is a bug requires this feature to exist.

bug or not i still think it is i cant click anything to make it right and it froze when i tried to type `{`

i actually use Windows at the moment of writing this, and on brave it did the same thing, so i can delete all the characters
![image](https://github.com/codicocodes/speedtyper.dev/assets/97115044/e0e8a94f-07e8-4138-a744-1af4fcc0256b)
![image](https://github.com/codicocodes/speedtyper.dev/assets/97115044/b46d0ff7-b607-4a0e-a248-7c9da4d83f2e)

**Comment by Jirafey** [comment]
![image](https://github.com/codicocodes/speedtyper.dev/assets/97115044/7fe60d1a-ea80-4563-aad0-4c3c5f92bce5)
refreshing tests, doesnt remove the "incorrect character in input"?
i need to refresh the page for input to work again.
